pages/api.js

```diff
@@ -66,6 +76,70 @@ const API = ({ api, services }) => {
     resume: rate_limiting_resume
   } = rate_limiting || {};
 
+  const handleScroll = throttle(() => {
+    const currentVisibleAnchor = getVisibleAnchor();
```

Comment:
I'd rather define function _before_ using it

Reply:
Good point :)